editor/editor_themes.cpp

```diff
@@ -702,6 +702,19 @@ Ref<Theme> create_editor_theme(const Ref<Theme> p_theme) {
 	theme->set_color("icon_focus_color", "Button", icon_focus_color);
 	theme->set_color("icon_pressed_color", "Button", icon_pressed_color);
 
+	// Variation for Editor Log filter buttons
+	theme->set_type_variation("editor_log_filter_button", "Button");
```

Comment:
```suggestion
	theme->set_type_variation("EditorLogFilterButton", "Button");
```
Type variations are like sub-types, so they should be capitalized accordingly.